scss/_settings_colors.scss

```diff
@@ -39,6 +39,7 @@ $active-background-opacity-amount: 0.15;
 
 // NON-SEMANTIC COLOURS
 $color-label-validated: #006b75;
+$color-code-background: rgba($color-x-dark, 0.03);
```

Comment:
Maybe we should move the other one (code snippet header background) here as well?